lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -130,6 +130,14 @@ def observed_dl1_files(temp_dir_observed_files):
         "2516351600",
     )
 
+    run_program(
+            "lstchain_check_dl1",
+            "-b",
+            "--omit-pdf",
+            "--input-file",
+            dl1_output_path1
+    )
+
```

Comment:
Still failing. You probably need to give the output path `temp_dir_observed_files`

Reply:
Yes, this was it, thanks! should be ok now.